content/concepts/consensus-network/amendments/known-amendments.md

```diff
@@ -552,6 +553,45 @@ This amendment fixes a bug in the [NonFungibleTokensV1][] amendment code where N
 
 This amendment has no effect unless the [NonFungibleTokensV1][] amendment is enabled. This amendment is obsolete because its effects are included as part of [NonFungibleTokensV1_1][].
 
+## fixNonFungibleTokensV1_2
+[fixNonFungibleTokensV1_2]: #fixnonfungibletokensv1_2
+
+| Amendment | fixNonFungibleTokensV1_2 |
+|:----------|:-----------|
+| Amendment ID | 73761231F7F3D94EC3D8C63D91BDD0D89045C6F71B917D1925C01253515A6669 |
+| Status | Open for Voting  |
+| Default Vote (Latest stable release) | No |
+| Pre-amendment functionality retired? | No |
+
+Amendment `fixNonFungibleTokensV1_2` is a combination of bug fixes that have been individually merged into feature/nft-fixes through the pull request process:
+
+**Fix Unburnable NFT**
+
+Currently, an NFT cannot be burned when it has over 500 offers. To remove this restriction, this change deletes exactly 500 offers, leaving any remaining offers untouched.
```

Comment:
@kennyzlei This doesn't necessary have to do with the `lsfBurnable` flag. What you described is one of scenarios(probably the most concerning among all though) that my pull request changes. But it might be good to make a generalization of the change first before talking about this specific situation. 

Generalized description:
`Currently, an NFT cannot be burned when it has over 500 offers. To remove this restriction, this change deletes exactly 500 offers upon burning the NFT, leaving any remaining offers untouched. `
In addition to the above, we can add another sentence to describe the biggest issue(among many others) that it solves. Something like:
`This addresses a concern where the issuer account cannot burn an NFT that has the `lsfBurnable` flag enabled, due to the exceeding number of offers.`

Reply:
@shawnxie999 I was trying to highlight how an NFT with 500 or less offers cannot be burned if the `lsfBurnable` flag is disabled, but maybe that is assumed knowledge of what the current state is.  If you feel like that is assumed, we can change it back

Reply:
cool, im fine with leaving it as it is now. Does make sense to highlight the major concern there

Reply:
@shawnxie999 No I think your suggestion is actually more clear, let's go with your suggestion